<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoUserDialog</class>
 <widget class="QDialog" name="InfoUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: '#0d3d4c';
}</string>
  </property>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>70</y>
     <width>201</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: '#101014';
	border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 30pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Multi Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Mutes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 11pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="mutesBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 12pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 12pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Is Muted</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mutedLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(125, 0, 0);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
